packages/formation-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/formation-react",
-  "version": "5.2.6",
+  "version": "5.2.7",
```

Comment:
I'm always curious about versioning, and teammates' opinions how to bump enhancements. In this case, I feel it might be more of a minor than a fix because of the updated focus method and new prop. Not a huge deal, but maybe worth calling this version 5.3.0.